### PR TITLE
[doc][minor] Updating the softmax benchmarks to a more modern pytorch + gpu

### PR DESCRIPTION
**Pull request by blefaudeux**
## What does this PR do?
The softmax curves that we had in the README/repo were with a V100 and pytorch 1.10, updating that to a more modern GPU (laptop 3080) and pytorch 1.10

## Before submitting

- [x] Did you have fun?
  - Make sure you had fun coding 🙃
- [x] Did you read the [contributor guideline](https://github.com/facebookresearch/fairscale/blob/master/CONTRIBUTING.md)?
- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
  - [x] N/A
- [x] Did you make sure to update the docs?
  - [ ] N/A
- [ ] Did you write any new necessary tests?
  - [x] N/A
- [ ] Did you update the [changelog](https://github.com/facebookresearch/fairscale/blob/master/CHANGELOG.md)? (if needed)
  - [x] N/A

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

### CODE REVIEW

**Comment by blefaudeux** [comment]
Note that the register spilling issues with the V100 (when testing on really big buffers, 100B-scale) are mostly gone

**Review by dianaml0 — APPROVED** [review]
